catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -39,28 +38,21 @@
 /**
  * Deletes records by ID.
  */
+@Service
 @Command(scope = CatalogCommands.NAMESPACE, name = "remove", description = "Deletes a record from the Catalog.")
-public class RemoveCommand extends CatalogCommands {
+public class RemoveCommand extends CqlCommands {
```

Comment:
During heroing and looking at the `--help`, since this command takes a list of `Metacard` IDs as input, I'm not sure it makes any sense to have CqlOptions associated with it.